tests/constructDummyData.js

```diff
@@ -0,0 +1,38 @@
+// This file builds dummy data objects for each schema, based on the fields.
```

Comment:
so do we have to run this before running the tests if, say, we added a new field ?

Reply:
It is not 100% necessary, as most validation scripts are not using the value from each field. But, yes, if the schema gets changed then it's a good idea to rerun the script to update the dummy files.

Reply:
so my concern is that these tests will always pass  because the fields each script needs are added in the test code, which will not detect people removing a field that the script uses, I'd say the script should lookup the fields from the dummy object and populate it, but throw an error if the field doesn't exist

Reply:
implemented in [new commit
](https://github.com/icgc-argo/argo-dictionary/pull/45/commits/7471c5bc8ebab986ba526477542298906fa35040)